config/zones/NA.yaml

```diff
@@ -77,3 +77,5 @@ fallbackZoneMixes:
         unknown: 0.0
         wind: 0.015505325244864603
 timezone: Africa/Windhoek
+region: Africa
+country: null
```

Comment:
Nulll?

Reply:
I decided to use `null` instead of `Other` for zones with an ambiguous country. It might potentially be a sensitive topic, so I thought it was the most appropriate to leave the value as null in these cases. Please bring other suggestions if you have any!

Reply:
I think that's a nice approach but this is Namibia and as far as I know it's not contested?